packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.js

```diff
@@ -107,6 +102,20 @@ export default class QuanticCaseClassification extends LightningElement {
    * @defaultValue `'Select an option'`
    */
   @api messageWhenValueMissing = this.labels.selectOption;
+  /**
+   * Whether the Quantic Case Classification component should automatically fetch new case classifications when its value changes.
```

Comment:
we are not really used to mention the name of the component in the property descriptions.
```suggestion
   * Indicates whether the component should automatically fetch new case classifications when its value changes.
```

Reply:
ok noted!